### DISCUSSION TITLE
misson doesn't get triggered

### DISCUSSION THREAD

**Discussion by daggs1** [endless-sky/endless-sky#9494] [General]
Greetings,

I'm working on a small storyline that is triggered by a specific case of an existing job in the game, here is the existing job: https://github.com/endless-sky/endless-sky/blob/master/data/remnant/remnant jobs.txt#L112 and here is what I'm trying to do:
```
mission "test: Expanded Horizons Astral job"
        invisible
        name "Research Expedition to <waypoints>"
        description "Travel to <waypoints> with a Research Laboratory to collect data, then return to <destination>."
        source
                government "Remnant"

        waypoint "Al Hurr"

        to offer
                has "remnant: ssil vida deactivation"
                has "remnant: ssil vida activation"
                has "Remnant: Expanded Horizons Astral 2: done"
                random < 25

        on offer
                require "Research Laboratory"

        on waypoint
                dialog "Your scientist unpacks the laboratory and takes the requested observations. A short while later, they have the data compiled on a set of data crystals and inform you that everything is ready for departure."
        on complete
                payment 500000
                "reputation: Remnant" ++
                dialog "When you land, a researcher quickly retrieves the data crystals with the information about the surveyed system and hands you the logistics allotment of <payment>."
                event "test: start" 30

```
I'm not getting any syntax error, I've tried entering the job center on different landing sites for 10 minutes and didn't saw the job.
I even matched the to offer conditions to the existing job and still no dice.
can anyone help me understand what am I doing wrong here?

Thanks

**Comment by bene-dictator** [comment]
Al Hurr is a hidden system, so it may have something to do with that.

Related: https://github.com/endless-sky/endless-sky/issues/8543

**Reply by daggs1** [reply]
in my saved game, Al Hurr is visible. in fcat I get the same behavior when I select a system in the gegno space too.
as part of my testing I saw that the same mission selects a system that is still hidden, any other system in the map including systems at korath space, the location of the Emerald Sword and even gengo space after they were revealed.

**Comment by TomGoodIdea** [comment]
you made the mission invisible ;)

**Reply by daggs1** [reply]
interesting, I need an event to make in visible then?

**Reply by TomGoodIdea** [reply]
I mean, you have the `invisible` attribute on the mission

**Reply by daggs1** [reply]
there are multiple missions which are marked the same but still get triggered, why?

**Reply by TomGoodIdea** [reply]
I think `job` and `repeat` is what you're looking for: https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#introduction

**Reply by daggs1** [reply]
you might right, I'll drop the invisible and test, if no go will add job

Thanks
